CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Network Server will never attempt RX1 for devices with `Rx1Delay` of 1 second.
 - Improved efficiency of ADR MAC commands.
 - Gateway Configuration Server will use the default WebSocket TLS port if none is set.
+- Network and Application Servers now maintain application downlink queue per-session.
```

Comment:
This is under the wrong version.

Reply:
Good catch. Surprisingly, `git` is being too smart during rebase in cases where a new release was made after the `CHANGELOG` edit. This happens all the time.